tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -197,6 +209,51 @@ export class ScalarCardDataTable {
               selectedStepData.PERCENTAGE_CHANGE =
                 (closestEndPoint.y - closestStartPoint.y) / closestStartPoint.y;
               continue;
+            case ColumnHeaderType.STEP_AT_MAX:
+              if (!closestEndPointIndex) {
+                continue;
+              }
+              selectedStepData.STEP_AT_MAX = this.getMaxValueInRange(
+                datum.points,
+                closestStartPointIndex,
+                closestEndPointIndex,
+                true
+              ).step;
+              continue;
+            case ColumnHeaderType.STEP_AT_MIN:
+              if (!closestEndPointIndex) {
+                continue;
+              }
+              selectedStepData.STEP_AT_MIN = this.getMinValueInRange(
+                datum.points,
+                closestStartPointIndex,
+                closestEndPointIndex,
+                true
+              ).step;
+              continue;
+            case ColumnHeaderType.MEAN:
+              if (!closestEndPointIndex) {
+                continue;
+              }
+              selectedStepData.MEAN = this.getMean(datum.points);
+              continue;
+            case ColumnHeaderType.REAL_CHANGE:
```

Comment:
There is inconsistency in column names. 

In the original table that appears when hovering over the chart, there is "smoothed" and "value", which I assume correspond to point.y and point.value respectively.

In the data table you also have SMOOTHED and VALUE, which is calculated with point.y and point.value. So far so good.

But you also have START_VALUE, END_VALUE, MIN_VALUE, MAX_VALUE, and VALUE_CHANGE. All calculated with point.y. That's inconsistent. Should they be START_SMOOTHED, END_SMOOTHED, MIN_SMOOTHED, MAX_SMOOTHED, and SMOOTHED_CHANGE? Similarly, should the user-visible names have the word Smoothed in them?

Now you are adding REAL_CHANGE, calculated with point.value. You are supposedly introducing "REAL" as a placeholder for what should be "VALUE". Thats inconsistent. Should this actually be VALUE_CHANGE.

And it appears that you intend to add NONSMOOTHED_START_VALUE and NONSMOOTHED_END_VALUE, adding more inconsistency. Should those just be START_VALUE and END_VALUE?

Reply:
Ok, so if I understand correctly, REAL_CHANGE should actually be RAW_CHANGE? If so, I think you can at least make that change in this PR since the field is new.